basic/list2.py

```diff
@@ -13,17 +13,23 @@
 # so [1, 2, 2, 3] returns [1, 2, 3]. You may create a new list or
 # modify the passed in list.
 def remove_adjacent(nums):
```

Comment:
:) not really. how about a test like: [1,1,2,1]?
it should return 1, 2, 1 but in your solution it return only  1, 2